README.md

```diff
@@ -5,32 +5,33 @@
 Convert from Tensorflow to CoreML
 =================================
 
-Python package to convert from Tensorflow to CoreML format. To get the latest
-version of `tfcoreml`, please run:
+[coremltools](https://github.com/apple/coremltools) (Recommended approach)
+--------------------
+
+
+For converting TensorFlow models to CoreML format, the recommended approach is to use `coremltools 4.0` python package.
+
+Please see the  coremltools documentation on [Tensorflow conversion](https://coremltools.readme.io/docs/tensorflow-conversion) for example usage.
+
+To get the latest version of coremltools:
 
 ```shell
-pip install --upgrade tfcoreml
+pip install coremltools==4.0b1
 ```
 
-For the latest changes please see the [release
-notes](https://github.com/tf-coreml/tf-coreml/releases).
+For the latest changes in coremltools, please see the [release notes](https://github.com/apple/coremltools/releases/).
```

Comment:
This also can be removed. We have pointed to the documentation, which has release notes as well

Reply:
Removed.